book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_reader :rental
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
+  end
+end
```

Comment:
- [ ] For your `Book` class you should have a method for adding rentals, the book itself should be sent as a parameter to create the new rental achieving the `has-many` association
Your code should look something like this :point_down: 

```ruby
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```